templates/config.txt

```diff
@@ -60,8 +60,6 @@ dtparam=audio=on
 #dtoverlay=vc4-kms-v3d
 #max_framebuffers=2
 
-[pi4]
-[cm4s]
 dtoverlay=dwc2,dr_mode=host
 
```

Comment:
```suggestion
```
We will add the dwc2 to the overlay. As this change would only affect the new images. But we need to fix this for all devices which run 5.10.

Reply:
Please remove the `dtoverlay=dwc2,dr_mode=host` we will add these to the revpi overlays.